File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex12.java
//Importando Scanner
import java.util.Scanner;

public class Ex12 {

	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double salarioFuncionario, salarioMinimo, quantidadeSalMinimo;
	
	System.out.println("Quantidaded de salários mínimos\n\n");
	
	//Entradas
	System.out.print("Insira o valor do salário mínimo: R$ ");
	salarioMinimo = ent.nextDouble();
	
	System.out.print("Insira o valor do salário do funcionáiro: R$ ");
	salarioFuncionario = ent.nextDouble();
	
	//Processamento
	quantidadeSalMinimo = salarioFuncionario / salarioMinimo;
	
	//Saída
	System.out.println("O funcionário ganha " + quantidadeSalMinimo + " salário(s) mínimos ");
		
	}
}

File: 3 semestre/Estrutura de Dados I/Listas/Lista de Matrizes/ex12.c
#include <stdio.h>

main()
{
    /* vars */
    int A[4][4], B[4][4], SOMA[4][4],  i, j;

    printf("SOMA DE DUAS MATRIZES A+B\n");

    /* Entradas */
    for (i = 0; i < 4; i++) {
        for (j = 0; j < 4; j++) {
            printf("Insira A[%d][%d]: ", i, j);
            scanf("%d", &A[i][j]);
        }
    }

    for (i = 0; i < 4; i++) {
        for (j = 0; j < 4; j++) {
            printf("Insira B[%d][%d]: ", i, j);
            scanf("%d", &B[i][j]);
        }
    }

    /* Processamento e Saída */
    printf("\nA + B:\n");
    for (i = 0; i < 4; i++) {
        for (j = 0; j < 4; j++) {
            SOMA[i][j] = A[i][j] + B[i][j];
            printf("%d  ", SOMA[i][j]);
        }
        printf("\n");
    }

}
